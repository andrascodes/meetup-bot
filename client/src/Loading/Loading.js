import React from 'react'

export const Loading = props => (
  <div className="container-fluid">
    <h2>Loading...</h2>
  </div>
)