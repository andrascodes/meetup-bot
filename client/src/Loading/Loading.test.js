import React from 'react'
import renderer from 'react-test-renderer'

import { Loading } from './Loading'

describe('Loading', () => {

  test('renders correctly', () => {
    let tree = renderer.create(
      <Loading />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

})