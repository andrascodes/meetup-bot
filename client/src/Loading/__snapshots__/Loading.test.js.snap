exports[`Loading renders correctly 1`] = `
<div
  className="container-fluid">
  <h2>
    Loading...
  </h2>
</div>
`;
